torch/_inductor/config.py

```diff
@@ -229,11 +229,13 @@ def is_fbcode():
 force_same_precision = (
     True if is_fbcode() else os.environ.get("TORCHINDUCTOR_FORCE_SAME_PRECISION") == "1"
 )
+
 # Specify candidate backends for gemm autotune.
-# Possible choices are combinations of: ATen, Triton, CUTLASS, CPP.
+# Possible choices are combinations of: ATen, Triton, CUTLASS, CK, CPP.
 # ATen: default Pytorch ATen kernels.
 # Triton: Triton templates defined in torch inductor.
 # CUTLASS: Cutlass templates and kernels.
+# CK: Composable Kernel templates and kernels.
```

Comment:
Nit: make backend / hardware mapping clear. e.g.
Triton: NVidia and AMD GPUs.
CUTLASS: NVidia GPUs only.
CK: AMD GPUs only.